src/features/offer/api/useSimilarOffers.native.test.ts

```diff
@@ -136,6 +140,23 @@ describe('useSimilarOffers', () => {
         )
       })
     })
+
+    it('should not call similar offers API when offer id provided, user share his position and shouldFetch is false', () => {
```

Comment:
ce nom `shouldFetch is false` me semble très technique et lié au détail d'implémentation, ça me fait penser qu'il serait peut etre bien d'ajouter un test à un plus haut niveau